Makefile

```diff
@@ -114,9 +111,7 @@ clean-all-boards:
 test-doc:
 	@make -C $(DOC_DIR) clean;\
 	make -C $(DOC_DIR) all DOC=$(DOC);\
-	if [ "`diff -q $(DOC_DIR)/$(DOC).aux $(DOC_DIR)/$(DOC).expected`" ]; then \
-	echo DOC $(DOC) TEST FAILED; else \
-	echo DOC $(DOC) TEST PASSED; fi >> test_report.log
+	diff -q $(DOC_DIR)/$(DOC).aux $(DOC_DIR)/$(DOC).expected
 
 test-all-docs:
 	$(foreach b, $(DOC_LIST), make test-doc DOC=$b;)
```

Comment:
remove foreach (prevents jenkins from detecting if the first build breaks), and replace with 2 make test-doc targets, one for each document type.